src/content/docs/en/guides/internationalization.mdx

```diff
@@ -456,3 +537,10 @@ console.log(getLocaleByPath("portugues")) // will log "pt-AO"
 
 [`site`]: /en/reference/configuration-reference/#site
 [`i18n.locales`]: /en/reference/configuration-reference/#i18nlocales
+
+
+### `redirectToDefaultLocale`
```

Comment:
@ematipico We're going to need some API reference for these new things! Can you take a first shot at adding reference descriptions for these?

Reply:
sure, will do